frontend/src/scenes/pipeline/BatchExportBackfill.tsx

```diff
@@ -0,0 +1,74 @@
+import { useActions, useValues } from 'kea'
+import { Form } from 'kea-forms'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonCalendarSelectInput } from 'lib/lemon-ui/LemonCalendar/LemonCalendarSelect'
+import { LemonField } from 'lib/lemon-ui/LemonField'
+import { LemonModal } from 'lib/lemon-ui/LemonModal'
+
+import { batchExportRunsLogic, BatchExportRunsLogicProps } from './batchExportRunsLogic'
+
+export function BatchExportBackfill({ id }: BatchExportRunsLogicProps): JSX.Element {
+    const logic = batchExportRunsLogic({ id })
+
+    const { batchExportConfig, isBackfillModalOpen, isBackfillFormSubmitting } = useValues(logic)
+    const { closeBackfillModal } = useActions(logic)
+
+    return (
+        <LemonModal
+            title="Backfill batch export"
```

Comment:
This used to be "Create historical export". We mix up "historical export" and "backfill" all over the place, so in this PR I've tried to use "backfill" everywhere.